pkg/lib/cockpit-components-listing-panel.jsx

```diff
@@ -26,6 +26,7 @@ import './cockpit-components-listing-panel.scss';
  *     - name tab name (has to be unique in the entry, used as react key)
  *     - renderer react component
  *     - data render data passed to the tab renderer
+ *     - hide flag whetever tab link and it's content should be shown
```

Comment:
This is hard to understand. "whetever" == "whether" or "whenever"? (the former would make more sense); and "should *not* be shown"?; also "it's" → "its".

Perhaps:

    hide: when true, hide tab link and its contents

Please also add ":" between the name and the description while you are touching this; it's confusing to read right now.